### DISCUSSION TITLE
How to run the crawler?

### DISCUSSION THREAD

**Discussion by Ehsan-U** [apify/crawlee-python#489] [Q&A] [answered]
Is there a way to run the entire project instead of running individual files with `python3 file.py`, similar to how Scrapy uses `scrapy crawl <spider>`? Does Crawlee (Python) have a command like this?
I couldn't find any reference in the Crawlee Python documentation on how to run the entire project.

**Comment by Ehsan-U** [comment]
Project Structure:

UP-SCRAPER/
│
├── crawler/
│   ├── crawler/
│   │   ├── \_\_init__.py
│   │   ├── \_\_main__.py
│   │   └── routes.py
│
├── poetry.lock
├── pyproject.toml
└── README.md

**Comment by vdusek** [comment, marked as answer]
Hi @Ehsan-U, Crawlee provides CLI, but in Python, it is now only for creating new projects via prepared [templates](https://github.com/apify/crawlee-python/tree/master/templates). For more information see [Setting up | With Crawlee CLI](https://crawlee.dev/python/docs/introduction/setting-up#with-crawlee-cli) doc page.
